metadata-transformer/src/main/java/no/unit/nva/metadata/NoLowercaseDCPrefixInMetadataRule.java

```diff
@@ -0,0 +1,30 @@
+package no.unit.nva.metadata;
+
+import org.apache.any23.validator.*;
+import org.w3c.dom.Node;
+
+import java.util.List;
+
+public class NoLowercaseDCPrefixInMetadataRule implements Rule {
+
+    @Override
+    public String getHRName() {
+        return "no-lowercase-dc-prefix-in-metadata-rule";
+    }
+
+    @Override
+    public boolean applyOn(DOMDocument document, RuleContext<?> context,
+                           ValidationReportBuilder validationReportBuilder) {
+        List<Node> metas = document.getNodes("/HTML/HEAD/META");
+        for (Node meta : metas) {
+            Node nameNode = meta.getAttributes().getNamedItem("name");
```

Comment:
I think what you are doing here is in pseudocode:
```
if( existsNodeWithLowerCaseDcPrefix(metas)){
reportError();
return true;
}

return false;
```

Reply:
I think if you refactor the code it will describe the intention better.

Reply:
I think the intent is pretty clear, but then again I just wrote it, and it might not be as clear for a theoretical dev comes across this not knowing what the Any23 Validator is. Writing it out like you specified I don't actually find the code to be much more readable, we've just added a descriptive method name, and honestly, the class name should have gotten the point across. I'm not sure I'd consider this worth 5 more lines, and doubling the amount of return statements, but maybe that's just me. I'll mess about with it some more.